contracts/test/MockL1ScrollMessanger.sol

```diff
@@ -43,9 +43,10 @@ contract MockL1ScrollMessenger is IL1ScrollMessenger {
 		bytes32 _xDomainCalldataHash = keccak256(
 			_encodeXDomainCalldata(_from, _to, _value, _nonce, _message)
 		);
+		// solhint-disable-next-line gas-custom-errors
 		require(
 			!isL2MessageExecuted[_xDomainCalldataHash],
-			"Message was already successfully executed"
+			"it was already successfully"
```

Comment:
To maintain compatibility with ScrollMessenger, please do not modify the messages.